src/components/Staking/StakingProductsCardGrid.tsx

```diff
@@ -272,7 +286,8 @@ const StakingProductCard: React.FC<ICardProps> = ({
         </List>
       </Box>
       <Box {...PADDED_DIV_STYLE}>
-        <ButtonLink to={url} customEventOptions={matomo} width="100%">
+        {/* TODO: Matomo - re-enable */}
+        <ButtonLink to={url} /* customEventOptions={matomo} */ width="100%">
```

Comment:
Oh interesting. So, we removed that `customEventOptions` from the Link...will create a card for that. Good catch.